scala/scala.bzl

```diff
@@ -50,7 +51,9 @@ def scala_specs2_junit_test(name, **kwargs):
 
 # Re-export private rules for public consumption
 scala_binary = _scala_binary
-scala_doc = _scala_doc
+make_scala_doc_rule = _make_scala_doc_rule
```

Comment:
is this to be expected to be used by user later? if not, can be removed I think.

or is it intended to be used with potentially custom aspect later?

Reply:
we will probably need to use `make_scala_doc_rule` and the `scaladoc_intransitive_aspect` to build it internally. That's why they are exported.

Reply:
Thanks. Makes sense. If you don't mind, we can add a comment here, so it's more clear for folks later on. E.g. people may notice this is not used in the repo and it gets removed.